guardian/support-service-lambdas
# Add Config to payment-failure-comms

## What does this change?
Add code to fetch config in the payment-failure-comms lambda. Also adds a Failure object that will be expanded in the future.

## How to test
To test send a message to the API Gateway URL and looks at the logs to confirm that the config was read correctly.
